src/features/auth/components/Login.tsx

```diff
@@ -45,12 +54,92 @@ export const Login: FC = () => {
+      <div className="w-full max-w-sm rounded-lg border border-gray-200 bg-white p-4 shadow dark:border-gray-700 dark:bg-gray-800 sm:p-6 md:p-8">
+        <Form {...form}>
+          <form
+            onSubmit={handleSubmit(onEmailAndPassword)}
+            className="space-y-6 justify-self-center"
+          >
+            <h5 className="text-xl font-medium text-gray-900 dark:text-white">
+              Sign in to{" "}
+              <label
+                htmlFor=""
+                className="font-dancing-script text-3xl font-bold"
+              >
+                Blogify
+              </label>
+            </h5>
+            <FormField
+              control={form.control}
+              name="email"
+              render={({field}) => (
+                <FormItem className="mb-5">
+                  <FormLabel className="mb-2 block text-sm font-medium text-gray-900 dark:text-white">
+                    email
+                  </FormLabel>
+                  <FormControl>
+                    <Input
+                      placeholder="email@gmail.com"
+                      {...field}
+                      {..._register("email")}
+                      className="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-sm text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500"
+                    />
+                  </FormControl>
+                </FormItem>
+              )}
+            />
+            <FormField
+              control={form.control}
+              name="password"
+              render={({field}) => (
+                <FormItem className="mb-5">
+                  <FormLabel className="mb-2 block text-sm font-medium text-gray-900 dark:text-white">
+                    password
+                  </FormLabel>
+                  <FormControl>
+                    <Input
+                      type="password"
+                      placeholder="******"
+                      {...field}
+                      {..._register("password")}
+                      className="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-sm text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500"
+                    />
+                  </FormControl>
+                </FormItem>
+              )}
+            />
+            <button
+              className="w-full rounded-lg bg-blue-700 px-5 py-2.5 text-center text-sm font-medium text-white hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800 sm:w-auto"
+              onClick={onProvider(GoogleAuthProvider)}
+            >
+              Sign-in
+            </button>
+            <Link
```

Comment:
Link navigates user to the "to" prop
no need to add additional onClick for navigation